### ISSUE TITLE
已经添加测试账号，但是测试订阅还是显示国家地区不符合

### ISSUE THREAD

**Issue by kartyjoin** [android/play-billing-samples#550]
已经在google play console中把账号添加为测试卡，一次性商品购买可以成功，订阅可以选择测试卡，但是会提示国家地区不符合要求，这个该怎么解决？

**Comment by matale** [comment]
Are you in China? Google Play doesn't work in China.
If not then maybe you have not setup the country in the subscription in Play console it is under Base Plan you set country/region availability then clear Play store data and wait a few hours for changes to take affect.

**Comment by matale** [comment]
Don't think it matters what region is set as there is some blocking going on at the internet level, surprised you are able to do purchases. You will probably have to resort to some sort of VPN which might cause its own issues.

**Comment by kartyjoin** [comment]
> Don't think it matters what region is set as there is some blocking going on at the internet level, surprised you are able to do purchases. You will probably have to resort to some sort of VPN which might cause its own issues.

Indeed, we use VPN to access GooglePlay, and we have also done other tests. Paying for a subscription with a real credit card in the same network environment is successful, but the test card payment provided by GoogleBilling is still a reminder of the country. incompatible. From another point of view, the GoogleBilling test card method should not be limited, how can I solve this problem?

**Comment by tangxiangpi** [comment]
![111](https://user-images.githubusercontent.com/18000558/226246376-b6be3027-f6c3-4055-a52e-4af128696323.png)
我也遇到了这个问题。我是订阅和应用内商品都不行；同样的网络环境我其他的app就可以
